src/OpenTelemetry.Instrumentation.ElasticsearchClient/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * Updated OTel SDK package version to 1.2.0
 * Update minimum full framework support to net462
 * Requests that get an HTTP status code of 404 are not marked as an error span status
+* Add MaxDbStatementLength option with default of 4096
```

Comment:
It's generally a good idea to mention the PR numbers associated with the changes in the CHANGELOG.

Reply:
You could look at the OTel repo for reference: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md